tools/shared/src/main/scala/org/scalajs/core/tools/linker/Versioned.scala

```diff
@@ -0,0 +1,25 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js tools             **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2014, LAMP/EPFL   **
```

Comment:
Right. I think there are sbt plugins that enforce that this kind of stuff is consistent all over a codebase.